kurtosis-devnet/pkg/kurtosis/sources/deployer/wallets.go

```diff
@@ -63,7 +63,7 @@ func (d *Deployer) getKnownWallets(ctx context.Context, fs *EnclaveFS) ([]*Walle
 		knownWallets = append(knownWallets, &Wallet{
 			Name:       key.String(),
 			Address:    addr.Hex(),
-			PrivateKey: fmt.Sprintf("%x", sec.D),
+			PrivateKey: fmt.Sprintf("%064x", sec.D),
```

Comment:
```
data, err := crypto.FromECDSA(sec)
// handle err

hexutils.Bytes(data).String()
```

Using the geth utils, and bytes types rather than raw ECDSA big ints, will prevent formatting issues

Reply:
ok. note that the output will then be prefixed by 0x
It *shouldn't* matter, but then again neiter should the lack of leading 0s :)

Reply:
ok. note that the output will then be prefixed by 0x
It shouldn't matter, but then again neiter should the lack of leading 0s :)